AzureTable/Microsoft.DataTransfer.AzureTable/Source/AzureTableSourceAdapter.cs

```diff
@@ -25,10 +26,35 @@ public AzureTableSourceAdapter(IAzureTableSourceAdapterInstanceConfiguration con
             this.configuration = configuration;
 
             table = CloudStorageAccount.Parse(configuration.ConnectionString).CreateCloudTableClient().GetTableReference(configuration.Table);
+
+            tRequestOptions = new TableRequestOptions();
```

Comment:
Can you extract switch-case to a method and use object initializer syntax? Like this:

```
requestOptions = new TableRequestOptions
{
    LocationMode = ToAzureLocationMode(configuration.LocationMode)
};
```